src/Language/SystemVerilog/AST/Number.hs

```diff
@@ -354,9 +354,7 @@ instance Show Number where
     show (UnbasedUnsized bit) =
         '\'' : show bit
     show (Decimal (-32) True value) =
-        if value < 0
```

Comment:
I'm not sure why this check is needed, but I couldn't get `RawNum (-1)` to work with it. So I just removed it